include/astshim/FrameDict.h

```diff
@@ -105,7 +105,7 @@ class FrameDict : public FrameSet {
 
     @throws std::invalid_argument if two Frames in the FrameSet have the same non-empty domain.
     */
-    explicit FrameDict(FrameSet const &frameSet) : FrameSet(std::move(*frameSet.copy())), _domainIndexDict() {
+    explicit FrameDict(FrameSet const &frameSet) : FrameSet(frameSet), _domainIndexDict() {
```

Comment:
I added it after adding C++ copy constructor and before adding Python support for copy constructor. So it's reasonable, but I agree it would be clearer to swap the last two

Reply:
I could combine the C++ and python copy constructor commits; I was basically waiting to see if there would be complaints about having the C++ copy constructors exposed to Python (but I prefer that if only to unit-test it!)

Reply:
I am fine with it being exposed to python. Swap if you can, otherwise it is ok I guess